package.json

```
@@ -101,7 +101,7 @@
     "@lit-labs/react": "^1.1.1",
     "@oddbird/popover-polyfill": "0.2.2",
     "@primer/behaviors": "^1.3.4",
-    "@primer/octicons-react": "^19.7.0",
+    "@primer/octicons-react": "19.8.0",
```

Hm somehow installing with npm fixed the version, but we want to keep the `^`

```suggestion
    "@primer/octicons-react": "^19.8.0",
```